tests/events/events_integration_test.py

```diff
@@ -0,0 +1,74 @@
+from checkout_sdk.events.events import RetrieveEventsRequest
+from tests.checkout_test_utils import assert_response, retriable
+from tests.payments.payments_test_utils import make_card_payment
+from tests.webhooks.webhooks_integration_test import register_webhook
+
+__PAYMENT_EVENT_TYPES = ['payment_approved',
+                         'payment_canceled',
+                         'payment_capture_declined',
+                         'payment_capture_pending',
+                         'payment_captured',
+                         'payment_chargeback',
+                         'payment_declined',
+                         'payment_expired',
+                         'payment_paid',
+                         'payment_pending',
+                         'payment_refund_declined',
+                         'payment_refund_pending',
+                         'payment_refunded',
+                         'payment_retrieval',
+                         'payment_void_declined',
+                         'payment_voided']
+
+
+def test_should_retrieve_default_event_types(default_api):
```

Comment:
Name should be ` test_should_retrieve_event_types` since this test is testing different scenarios (default, v1, v2, etc).